mission-control/docs/reference/scripting/cel.md

```diff
@@ -19,7 +18,108 @@ spec:
 ```
 
 ---
-## Collections
```

Comment:
This should go into canary-checker, the Makefile copies the canary scripting file to mission control docs